### PR TITLE
Colors: Initial pass on updated palette.

### PR DESCRIPTION
**Pull request by sfrisk**
Need to fix a spacing issue, but basic pass at the colors.  A codepen illustrating this palette can be found at http://codepen.io/sfrisk/pen/bbeec265547b3554d047c445da957c1c/

### CODE REVIEW

**Comment by arschmitz** [comment]
:+1:  the colors look good i like them. Remember for previews we have view.css-chassis.com all you need to do is push to a branch in the main repo instead of your fork

**Comment by sfrisk** [comment]
Yeah, it was more I didn't have a visual demo of these yet.

File: scss/variables/colors.js [hunk 1]
```diff
@@ -8,41 +8,64 @@
...
+				"darker": "#D32F2F"
+			}
 	},
-	"chassis-blue": {
-		"value": "#4fc0c8",
-		"name": "Chassis Blue"
+	"text": {
+			name: "Text Colors",
+			value: {
+				"base": "#212121",
```
Comment (geekman-rohit) [inline scss/variables/colors.js]: too light? maybe something darker like `#111111` ?

File: scss/variables/colors.js [hunk 2]
```diff
@@ -8,41 +8,64 @@
...
+				"light": "#727272"
+			}
```
Comment (geekman-rohit) [inline scss/variables/colors.js]: fails WCAG AAA for normal size text.
Something like `#555555` would pass all tests

**Comment by geekman-rohit** [comment]
I think the colors are good. But they could be a little darker to pass all WCAG tests.
I suggest the following:
- The base colors should pass all tests
- The lighter ones may be allowed to fail the AAA tests for normal size but must pass all other
- Colors in palettes like primary, success to be checked as background, with a white foreground

**Comment by cvrebert** [comment]
Have y'all considered testing the distinguishability of the colors to people who are colorblind?

**Comment by arschmitz** [comment]
@cvrebert more then thought about https://github.com/jquery/css-chassis/pull/114

**Comment by cvrebert** [comment]
Ah, didn't know axe covered that. Neat.

**Comment by sfrisk** [comment]
Updated colors to reflect what is in the button PR. If that looks good, I'll merge it.

**Pull request closed without merging**